.github/workflows/test-linux-cpu.yml

```diff
@@ -0,0 +1,57 @@
+name: Unit-tests on Linux CPU
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+  workflow_dispatch:
+env:
+  CHANNEL: "nightly"
+jobs:
+  tests:
+    name: "Unit-tests on Linux CPU"
+    runs-on: [self-hosted, linux.8xlarge]
+    container:
+      image: pytorch/conda-builder:cpu
+    strategy:
+      matrix:
+        py_vers: ["3.8"]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Set Release CHANNEL (for release)
+        if: ${{ (github.event_name == 'pull_request' && startsWith(github.base_ref, 'release')) || startsWith(github.ref, 'refs/heads/release') }}
+        run: |
+          echo "CHANNEL=test" >> "$GITHUB_ENV"
+      - name: Install TorchVision
```

Comment:
Nit: you might want to try out https://github.com/pytorch/test-infra/tree/main/.github/actions/setup-miniconda here, it would hide all the complex logic there like ENV_NAME I think

Reply:
Seeing such errors when using the setup-miniconda action: https://github.com/pytorch/vision/actions/runs/3252067256/jobs/5337858164. Perhaps this is due to conda-build being installed in the same conda-env? Reverting to the local conda env for now.